EntityGeneratorBundle/Doctrine/EntityTrait/Replacer.php

```diff
@@ -88,31 +88,47 @@ private function getMethodBody(
     ): string
     {
         $template = <<<'TPL'
-    $updatedEntities = [];
-        $fallBackId = -1;
-        foreach ([PARAM_NAME] as $entity) {
-            /** @var string|int $index */
-            $index = $entity->getId() ? $entity->getId() : $fallBackId--;
-            $updatedEntities[$index] = $entity;
+    foreach ([PARAM_NAME] as $entity) {
             $entity->[MAPPED_BY]($this);
         }
```

Comment:
is this properly indented?

Reply:
yes, it's required that way.
https://github.com/irontec/ivozprovider/pull/1922/commits/3ef25a3b79f4bc1f5cd1ba3cf9cdf030160f2aa4#diff-3191334d05ca72838521e948f7d7799883f14566508b3d84e001b78c23b2526fL136